src/quantuminspire/qiskit/backend_qx.py

```diff
@@ -188,7 +184,13 @@ def get_experiment_results(self, qi_job):
             histogram = QuantumInspireBackend.__convert_histogram(result, measurements, result['number_of_qubits'],
                                                                   job['number_of_shots'])
             histogram_obj = Obj.from_dict(histogram)
-            experiment_result_data = ExperimentResultData(counts=histogram_obj)
+            raw_data_url = result.get('raw_data_url')
+            raw_data = self.__api.get(raw_data_url)
+            memory_data = self.__get_memory_data(raw_data, measurements, result['number_of_qubits'])
```

Comment:
Another thought, now that we are accessing the raw data, we also have the count as well. So maybe converting the histogram in line 184 is an overkill? I.e., we convert our QI histogram into a number of counts, while now there is a method to get the actual counts.

Reply:
I think you are right but what to do with the exception nr_shots = 1. No raw data is available from the backend. We must agree on how to construct the histogram. Issue [DEM-766]

Reply:
This should be fixed in ticket DEM-766